modules/WES/domains/level_1.yaml

```diff
@@ -0,0 +1,179 @@
+name: Level1
+id: https://w3id.org/htan/wes/level_1
+description: Bulk Whole Exome Sequencing Level 1 - Raw files
+imports:
+  - linkml:types
+
+prefixes:
+  htan: https://w3id.org/htan/
+  linkml: https://w3id.org/linkml/
+  NCIT: http://purl.obolibrary.org/obo/NCIT_
+
+default_prefix: htan
+
+enums:
```

Comment:
Do we have a design norm for how we order enums? Is this currently in order they appeared in the RFC / source data model? I think being explicit and consistent would be good. I would possibly advocate for alphabetical. That might be possible to automate with a linter,